src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -252,9 +265,9 @@ public function getCondition()
     public function getDeliveryInformation()
     {
         if ($this->product['quantity'] > 0) {
-            return Configuration::get('PS_LABEL_DELIVERY_TIME_AVAILABLE', $this->language->id);
+            return $this->configuration->get('PS_LABEL_DELIVERY_TIME_AVAILABLE')[$this->language->id];
```

Comment:
I can imagine scenarios where this will be undefined :)

Reply:
Fixed